app/views/clients/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <div class="row">
+    <div class="col-lg-3">
```

Comment:
in some places you use col-xs in some col-lg is used? why? 
and what should we use if we are only applying one of them.

Reply:
what did you fix?

Reply:
and where is my answer?